flink-table/flink-table-planner-blink/src/test/java/org/apache/flink/table/planner/factories/TestValuesTableFactory.java

```diff
@@ -619,7 +630,8 @@ private ChangelogMode parseChangelogMode(String string) {
                     SupportsFilterPushDown,
                     SupportsLimitPushDown,
                     SupportsPartitionPushDown,
-                    SupportsReadingMetadata {
+                    SupportsReadingMetadata,
+                    SupportsAggregatePushDown {
```

Comment:
Could you revert the changes on this class? Because it's hard to know whether it is implemented correctly before the planner supports `SupportsAggregatePushDown`.